src/pages/ImportWorkflow/ImportWorkflow.test.ts

```diff
@@ -21,33 +21,11 @@ type AppsContract = ReturnType<typeof Apps>;
 jest.mock('src/libs/ajax');
 jest.mock('src/libs/ajax/leonardo/Apps');
 
-type WorkspaceUtilsExports = typeof import('src/components/workspace-utils');
-jest.mock('src/components/workspace-utils', (): WorkspaceUtilsExports => {
-  const { h } = jest.requireActual('react-hyperscript-helpers');
-
-  const useWorkspaces = jest.fn();
+type UseWorkspacesExports = typeof import('src/workspaces/useWorkspaces');
+jest.mock('src/workspaces/useWorkspaces', (): UseWorkspacesExports => {
   return {
-    ...jest.requireActual('src/components/workspace-utils'),
-    useWorkspaces,
-    // WorkspaceImporter is wrapped in withWorkspaces to fetch the list of workspaces.
-    // withWorkspaces calls useWorkspaces.
-    // However, since withWorkspaces and useWorkspaces are in the same module, simply
-    // mocking useWorkspaces won't work: withWorkspaces will use the unmocked version.
-    // So we have to mock withWorkspaces.
-    // And since WorkspaceImporter calls withWorkspaces at the module level, it must
-    // be mocked here, not in a before... block.
-    // Thus, we also mock useWorkspaces to allow tests to control the returned workspaces.
```

Comment:
none of this extra wonkiness is needed now that useWorkspaces is off on it's own module that this test file can just mock directly.